relational_operators/HashJoinOperator.cpp

```diff
@@ -65,21 +65,11 @@ DEFINE_bool(vector_based_joined_tuple_collector, false,
             "If true, use simple vector-based joined tuple collector in "
             "hash join, with a final sort pass to group joined tuple pairs "
             "by inner block. If false, use unordered_map based collector that "
-            "keeps joined pairs grouped by inner block as they are found "
-            "(this latter option has exhibited performance/scaling problems, "
-            "particularly in NUMA contexts).");
+            "keeps joined pairs grouped by inner block as they are found.");
```

Comment:
Sure, I can change that. 
Do we actually need the vector_based collector at all? If it’s unclear whether it will ever outperform map_based, maybe we should just get rid of it? 


> On Jun 27, 2016, at 13:53, Harshad Deshmukh <notifications@github.com> wrote:
> 
> In relational_operators/HashJoinOperator.cpp <https://github.com/apache/incubator-quickstep/pull/41#discussion_r68634602>:
> 
> > @@ -65,21 +65,11 @@ DEFINE_bool(vector_based_joined_tuple_collector, false,
> >              "If true, use simple vector-based joined tuple collector in "
> >              "hash join, with a final sort pass to group joined tuple pairs "
> >              "by inner block. If false, use unordered_map based collector that "
> > -            "keeps joined pairs grouped by inner block as they are found "
> > -            "(this latter option has exhibited performance/scaling problems, "
> > -            "particularly in NUMA contexts).");
> > +            "keeps joined pairs grouped by inner block as they are found.");
> Hi @navsan <https://github.com/navsan>
> How about changing the flag name to map_based_joined_tuple_collector and switch the default to true?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/apache/incubator-quickstep/pull/41/files/258970235f2b2d62cb4faefeffe1d2a8ccd53bc1#r68634602>, or mute the thread <https://github.com/notifications/unsubscribe/ACZB65jQr1J2bVFXiC7GyiZ3em2eEWQgks5qQBwugaJpZM4I_RVv>.
>